### PR TITLE
Remove 'Digital Ocean' section

### PR DESCRIPTION
**Pull request by lucydodo**
Fix https://github.com/getredash/setup/issues/67
Reference: https://github.com/getredash/redash/discussions/6812

Since Redash has been removed from the 'Digital Ocean Marketplace', remove its entry from the article.

### CODE REVIEW

File: src/pages/kb/open-source/setup.md [hunk 1]
```diff
@@ -82,7 +77,7 @@ Now proceed to “[Setup](#setup-redash-instance-setup)”.
 
 ### <a name="other"></a> Other
 
-The AWS, DigitalOcean and Google Compute Engine images are created using our [Setup Script](https://github.com/getredash/setup), which is designed to run on Ubuntu 18.04 server. You can either use the script as is (if you intend to run it on Ubuntu) or use it as a blueprint to create your own setup.
+The AWS and Google Compute Engine images are created using our [Setup Script](https://github.com/getredash/setup), which is designed to run on Ubuntu 18.04 server. You can either use the script as is (if you intend to run it on Ubuntu) or use it as a blueprint to create your own setup.
```
Comment (justinclift) [inline src/pages/kb/open-source/setup.md:80]: We should probably update the Ubuntu 18.04 reference while we're updating the text in this sentence.  Maybe replace that bit with something like:

```
... which is designed to run on many modern Linux distributions.  You can either use
the script as is, or use it as a blueprint to create your own setup.
```

Comment (lucydodo) [reply src/pages/kb/open-source/setup.md:80]:
```suggestion
The AWS and Google Compute Engine images are created using our [Setup Script](https://github.com/getredash/setup), which is designed to run on many modern Linux distributions. You can either use the script as is, or use it as a blueprint to create your own setup.
```

That's a good point, is this what you're saying?

**Review by justinclift — APPROVED** [review]
Cool, looks good to me. :smile:

File: src/pages/kb/open-source/setup.md (hunk 1 above)
Comment (justinclift) [reply src/pages/kb/open-source/setup.md]: Cool, I didn't realise it was possible to add suggestions directly like that + have them incorporated into PRs.  That's useful. :smile:

**Comment by justinclift** [comment]
Thanks heaps for getting this done @lucydodo. :smile:

**Pull request merged**